files/zh-cn/web/css/list-style-type/index.md

```diff
@@ -67,88 +65,143 @@ list-style-type 的属性值可以是以下任意一项：
   - : 实心方块
 - `decimal`
   - : 十进制阿拉伯数字
+    - 从 1 开始
```

Comment:
There is some convertion errors on this file, so I just modify it (consist with the english document)

```suggestion
```